grid-proxy/docs/database.md

```diff
@@ -0,0 +1,11 @@
+# Database
+
+The grid proxy has access to a postgres database containing information about the tfgrid.\
+The grid proxy mainly retrieves information from the db (maybe with a few modifications for efficient retrieval)\
```

Comment:
How is that database is populated, what are the few modifications re we talking about? denormalization, indices, .. etc?